config/imsim-config.yaml

```diff
@@ -25,7 +25,8 @@ eval_variables:
 # updated for each output file.
 # Also includes things that need some set up at the start of an exposure, like the atmospheric PSF.
 input:
-    camera_geometry: {}
+    camera_geometry:
+        camera_class: lsst.obs.lsst.LsstCam
```

Comment:
I don't love having the full specification of the lsst stack namespace here.  Can we make this just be `LsstCam`?  That seems like the extent of what the user should have to specify.

Reply:
That's fine, but then we'd have to prepend the `lsst.obs.lsst` part in a few different places.

Reply:
Right.  I guess my point is that that should be our job, not the user's job.

Reply:
@rmjarvis I think I've addressed your concerns.  Please have another look.